[2024-10-04T02:44:34.432+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-04T02:44:34.459+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_data_dag.extract manual__2024-10-04T02:44:32.037243+00:00 [queued]>
[2024-10-04T02:44:34.474+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_data_dag.extract manual__2024-10-04T02:44:32.037243+00:00 [queued]>
[2024-10-04T02:44:34.475+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-04T02:44:34.494+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): extract> on 2024-10-04 02:44:32.037243+00:00
[2024-10-04T02:44:34.505+0000] {standard_task_runner.py:72} INFO - Started process 237 to run task
[2024-10-04T02:44:34.514+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stock_data_dag', 'extract', 'manual__2024-10-04T02:44:32.037243+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmphdsvnq91']
[2024-10-04T02:44:34.519+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask extract
[2024-10-04T02:44:34.623+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_data_dag.extract manual__2024-10-04T02:44:32.037243+00:00 [running]> on host 2a293f7fa826
[2024-10-04T02:44:34.737+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stock_data_dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2024-10-04T02:44:32.037243+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-04T02:44:32.037243+00:00'
[2024-10-04T02:44:34.740+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-04T02:44:34.744+0000] {logging_mixin.py:190} INFO - Start extracting
[2024-10-04T02:44:34.747+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=172.17.0.1:9092 <connecting> [IPv4 ('172.17.0.1', 9092)]>: connecting to 172.17.0.1:9092 [('172.17.0.1', 9092) IPv4]
[2024-10-04T02:44:34.748+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2024-10-04T02:44:34.749+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=172.17.0.1:9092 <connecting> [IPv4 ('172.17.0.1', 9092)]>: Connection complete.
[2024-10-04T02:44:34.858+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2024-10-04T02:44:34.859+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2024-10-04T02:44:34.860+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2024-10-04T02:44:34.963+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2024-10-04T02:44:34.963+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2024-10-04T02:44:34.977+0000] {conn.py:380} INFO - <BrokerConnection node_id=0 host=instance-ubuntu-2204-minikube.asia-east1-c.c.modern-environs-436110-t9.internal:9092 <connecting> [IPv4 ('10.0.0.10', 9092)]>: connecting to instance-ubuntu-2204-minikube.asia-east1-c.c.modern-environs-436110-t9.internal:9092 [('10.0.0.10', 9092) IPv4]
[2024-10-04T02:44:34.977+0000] {conn.py:1205} INFO - Probing node 0 broker version
[2024-10-04T02:44:34.985+0000] {conn.py:410} INFO - <BrokerConnection node_id=0 host=instance-ubuntu-2204-minikube.asia-east1-c.c.modern-environs-436110-t9.internal:9092 <connecting> [IPv4 ('10.0.0.10', 9092)]>: Connection complete.
[2024-10-04T02:44:34.986+0000] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=172.17.0.1:9092 <connected> [IPv4 ('172.17.0.1', 9092)]>: Closing connection. 
[2024-10-04T02:44:35.091+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2024-10-04T02:44:35.092+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2024-10-04T02:44:35.099+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=172.17.0.1:9092 <connecting> [IPv4 ('172.17.0.1', 9092)]>: connecting to 172.17.0.1:9092 [('172.17.0.1', 9092) IPv4]
[2024-10-04T02:44:35.100+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2024-10-04T02:44:35.101+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=172.17.0.1:9092 <connecting> [IPv4 ('172.17.0.1', 9092)]>: Connection complete.
[2024-10-04T02:44:35.206+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2024-10-04T02:44:35.207+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2024-10-04T02:44:35.215+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/test_dag.py", line 59, in extract
    conn = psycopg2.connect(**db_params)
  File "/home/airflow/.local/lib/python3.10/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?

[2024-10-04T02:44:35.243+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=stock_data_dag, task_id=extract, run_id=manual__2024-10-04T02:44:32.037243+00:00, execution_date=20241004T024432, start_date=20241004T024434, end_date=20241004T024435
[2024-10-04T02:44:35.279+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-04T02:44:35.280+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 32 for task extract (connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
; 237)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/test_dag.py", line 59, in extract
    conn = psycopg2.connect(**db_params)
  File "/home/airflow/.local/lib/python3.10/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?

[2024-10-04T02:44:35.331+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-04T02:44:35.389+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-04T02:44:35.393+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
